source/Jobbr.Server.UnitTests/Core/JobServiceTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using AutoMapper;
+using Jobbr.ComponentModel.JobStorage.Model;
+using Jobbr.Server.Core;
+using Jobbr.Server.Core.Models;
+using Jobbr.Server.Storage;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Shouldly;
+
+namespace Jobbr.Server.UnitTests.Core
+{
+    [TestClass]
+    public class JobServiceTests
+    {
+        private readonly IJobService service;
+        private readonly Mock<IJobbrRepository> repositoryMock;
+
+        public JobServiceTests()
+        {
+            var config = new MapperConfiguration(cfg => cfg.AddProfile<ModelToStorageMappingProfile>());
+
+            var mapper = config.CreateMapper();
+
+            repositoryMock = new Mock<IJobbrRepository>();
+
+            service = new JobService(repositoryMock.Object, mapper);
```

Comment:
Since there's only a single test, you could also add it directly into the test itself, unless you plan to extend this test class